internals/utils/src/dev/templates/error.ts

```diff
@@ -0,0 +1,38 @@
+import encodeHTML from 'escape-html';
```

Comment:
I believe these `.ts` files are generated (from the `.jst` and `.tsdef` files) and should not be committed to git.

Reply:
Yes you're right. I need to fix the gitignore file